lib/livebook/intellisense.ex

```diff
@@ -138,11 +138,20 @@ defmodule Livebook.Intellisense do
 
   defp format_completion_item({:map_field, name, _value}),
     do: %{
-      label: name,
+      label: "#{name}",
       kind: :field,
       detail: "field",
       documentation: nil,
-      insert_text: name
+      insert_text: "#{name}"
+    }
+
+  defp format_completion_item({:in_struct_field, struct_name, name, default}),
+    do: %{
+      label: "#{name}",
+      kind: :field,
+      detail: "#{struct_name} struct field",
+      documentation: "Default: #{inspect(default)}",
```

Comment:
This supports markdown, so we can use an elixir code snippet for highlighting. You can bring `value_snippet` back from https://github.com/livebook-dev/livebook/pull/756/files, then maybe like:

```
"""
**Default**

#{value_snippet(default, @line_length)}
"""
```